vowpalwabbit/core/include/vw/core/reductions/slates.h

```diff
@@ -16,7 +16,7 @@ namespace VW
 {
 namespace reductions
 {
-class slates_data
+class slates_data_obj
```

Comment:
I'm not crazy about postfixing class names with `obj` I find it redundant and misleading (`slate_reduction` is the object `slates_data_obj` is the type)

Reply:
yea `slates_reduction` is fine, I have been renaming the data instance in setup to be `slate_data` which was causing the issue

Reply:
We should prefer naming for the type and if the object has to be named something weird then so be it. One of the unfortunate outcomes of using snake_case for type names...